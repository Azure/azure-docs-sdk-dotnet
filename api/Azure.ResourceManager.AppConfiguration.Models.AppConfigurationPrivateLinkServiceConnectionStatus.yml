### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus&text=AppConfigurationPrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.op_Equality(Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus,Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.op_Inequality(Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus,Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Equals(Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Disconnected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
commentId: T:Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
namespace: Azure.ResourceManager.AppConfiguration.Models
name: AppConfigurationPrivateLinkServiceConnectionStatus
fullName: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
nameWithType: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.AppConfiguration.dll
packagesWithMoniker:
- value: Azure.ResourceManager.AppConfiguration v1.0.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.AppConfiguration v1.1.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct AppConfigurationPrivateLinkServiceConnectionStatus : IEquatable<Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type AppConfigurationPrivateLinkServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure AppConfigurationPrivateLinkServiceConnectionStatus

      Implements IEquatable(Of AppConfigurationPrivateLinkServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private link service connection status.
metadata:
  api_name:
  - Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.AppConfiguration.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
  - Azure::ResourceManager::AppConfiguration::Models::AppConfigurationPrivateLinkServiceConnectionStatus
  - AppConfigurationPrivateLinkServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
