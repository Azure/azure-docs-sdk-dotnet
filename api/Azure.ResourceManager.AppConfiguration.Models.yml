### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationKeyVaultProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationNameAvailabilityContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationNameAvailabilityResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateEndpointConnectionReference
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationRegenerateKeyContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationStoreApiKey
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationStorePatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.ArmAppConfigurationModelFactory
  monikers:
  - azure-dotnet-preview
enums:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationCreateMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationActionsRequired
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPublicNetworkAccess
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationResourceType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.AppConfiguration.Models
commentId: N:Azure.ResourceManager.AppConfiguration.Models
name: Azure.ResourceManager.AppConfiguration.Models
fullName: Azure.ResourceManager.AppConfiguration.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.AppConfiguration.Models
  - Azure::ResourceManager::AppConfiguration::Models
  - Models
  - AppConfiguration.Models
  - AppConfiguration::Models
  helpviewer_keywords:
  - Azure.ResourceManager.AppConfiguration.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
