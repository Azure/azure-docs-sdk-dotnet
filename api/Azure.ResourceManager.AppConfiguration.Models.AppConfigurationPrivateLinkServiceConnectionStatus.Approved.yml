### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus&text=AppConfigurationPrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  nameWithType: AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As AppConfigurationPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.AppConfiguration.Models
name: Approved
fullName: Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
nameWithType: AppConfigurationPrivateLinkServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.AppConfiguration.dll
packagesWithMoniker:
- value: Azure.ResourceManager.AppConfiguration v1.0.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.AppConfiguration v1.1.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  - Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.AppConfiguration.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.AppConfiguration.Models.AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  - Azure::ResourceManager::AppConfiguration::Models::AppConfigurationPrivateLinkServiceConnectionStatus::Approved
  - Approved
  - AppConfigurationPrivateLinkServiceConnectionStatus.Approved
  - AppConfigurationPrivateLinkServiceConnectionStatus::Approved
  helpviewer_keywords:
  - AppConfigurationPrivateLinkServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct AppConfigurationPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
